PennyPincher.API/PennyPincher/Controllers/AnalysisBridgeController.cs

```diff
@@ -0,0 +1,243 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PennyPincher.Models;
+using System.Xml.Linq;
+
+namespace PennyPincher.Controllers
+{
+    [ApiController]
+    [Route("/api/analysis")]
+
+    public class AnalysisBridgeController : Controller
+    {
+        List<CashFlowDto> CurrItemLogsCF = CashFlowDataStore.CurrentItemLogCashFlow.CashFlowsList;
+        List<CashFlowDto> ProjectedCF = CashFlowDataStore.ProjectedCashFlow.CashFlowsList;
+
+        public enum CFDataStores
+        {
+            Current = 0,
+            Projected =1
+        }
+        private List<CashFlowDto> CFDataStorePicker(CFDataStores TargetDataStore)
+        {
+            List<CashFlowDto> CFData = new List<CashFlowDto>();
+            switch (TargetDataStore) // Switch case to easily analyze each datastore the same way & scalable for additional datastor
+            {
+                case CFDataStores.Current:
+                    CFData = CashFlowDataStore.CurrentItemLogCashFlow.CashFlowsList;
+                    break;
+                case CFDataStores.Projected:
+                    CFData = CashFlowDataStore.ProjectedCashFlow.CashFlowsList;
+                    break;
+                default:
+                    CFData = CashFlowDataStore.CurrentItemLogCashFlow.CashFlowsList;
+                    break;
+            }
+            return CFData;
+        }
+        private List<CashFlowDto> ConvertLogToCF(List<ItemDto> loggedItems)
+        {
+            List<CashFlowDto> CFLogs = new List<CashFlowDto>();
+            List<CategoryTypes> Expenses = GetExpenseCategoryTypes();
+            CFLogs.Clear();
+
+            if (loggedItems.Count == 0)
+            {
+                return CFLogs;
+            }
+            foreach (var item in loggedItems)
+            {
+                CashFlowDto newCF = new CashFlowDto()
```

Comment:
As discussed previously, this is fine to manually to update each property for now. I can see this however, becoming tedious when our data objects become more complex with more requirements. We will figure this out in the future on a better approach.